.circleci/config.yml

```diff
@@ -13,6 +15,14 @@ jobs:
           key: go-modules-{{ checksum "go.mod" }}
           paths:
             - /go/pkg/mod
-      - run: make testacc
+      - secrethub/exec:
+          step-name: Run acceptance tests
+          command: make testacc
+          # SecretHub credential to use in the acceptance tests
+          flags: -e SECRETHUB_CREDENTIAL=secrethub/terraform-provider/testacc/secrethub/credential
```

Comment:
Just flying by here, how does this work pointing to a path instead of the contents?

Reply:
`-e` just takes SecretHub paths, no `secrethub://` needed. 

To avoid potential confusion, https://github.com/secrethub/terraform-provider-secrethub/pull/83/commits/14ca082cae84511b1b1fba58d0aa82d9220b689b refactors this part to ditch the `-e` flag.

Reply:
Huh so are you telling me you're getting a credential from secrethub to then call secrethub?

Reply:
Indeed, that second credential is a different one than the first credential.

The first one is a simple service account that represents CircleCI setting everything up for the job and just needs read access like most CIs do.

The second one is a bit different: it's a service account with admin rights to control the _acceptance test sandbox_, which is a different workspace than the one that contains the static configuration secrets. So that second does lots of creating and destroying of resources every time the tests are triggered, because it needs to test everything the Terraform provider supports (i.e. secrets, AWS and standard service accounts, dirs, access rules) and has a very cluttered and meaningless audit log.

So long story short: 
- it's not stuff regular users would do, but that's because deploying/developing SecretHub with SecretHub is a thing only we do
- it needs to be documented somewhere

Reply:
And yes: this is probably something to brainstorm about (in person).

Reply:
> it needs to be documented somewhere

Very minimal, but there's: https://github.com/secrethub/terraform-provider-secrethub#testing